### ISSUE TITLE
VoiceOver is Not Announcing Pagination Links Correctly in Editors

### ISSUE THREAD

**Issue by MRayermannMSFT** [Microsoft/AzureStorageExplorer#205]
VoiceOver is announcing pagination links as "Link 1, link 2, link 3 , link 4, Link greater, link comma". These are not helpful to users who cannot see. We should write HTML that has VoiceOver give more relevant speech output.

Fixing this bug will close:
- https://msdata.visualstudio.com/Cosmos/_workitems/edit/120385
